### PR TITLE
Improved Contributors Section

### LINKED ISSUE #599: [style]: Improve UI of contributors page
### What's the style idea?

Improving UI of contributors page, to add animations and hover effect. Please assign me this issue under GSSOC'23. 

### Add screenshots

Add screenshots

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Vidip-Ghosh**
## Related Issue

Closes #599 

## Description

I have added a new color scheme to the contributors section of the website. 

## Screenshots

Before:

Screenshot 2023-06-13 at 22 43 41

After:

Screenshot 2023-06-13 at 22 40 27

## Checklist 

- [x] My code adheres to the established style guidelines of the project.
- [ ] I have included comments in areas that may be difficult to understand.
- [x] My changes have not introduced any new warnings.
- [x] I have conducted a self-review of my code.

### CODE REVIEW

**Comment by JasonDsouza212** [comment]
@Vidip-Ghosh, I believe you may be confused. If you click on the open-source community, you will be able to see all the people who have contributed. Therefore, I would like that page to be redesigned, not the one you have edited

**Comment by JasonDsouza212** [comment]
Screenshot_20230613-233030

**Comment by Vidip-Ghosh** [comment]
> @Vidip-Ghosh, I believe you may be confused. If you click on the open-source community, you will be able to see all the people who have contributed. Therefore, I would like that page to be redesigned, not the one you have edited

Okay I got it

**Comment by Vidip-Ghosh** [comment]
I have changed the color scheme. I changed into opacity from grayscale. 

**Before**
Screenshot 2023-06-14 at 19 48 12

**After**
Screenshot 2023-06-14 at 19 47 36

Please check if its alright.

**Comment by JasonDsouza212** [comment]
@Vidip-Ghosh, please try to maintain the same theme. The suggestion I can give is that you display the username in a card, but in a more organized and attractive manner.  make sure to explore all the data that the API call is returning and implement the necessary data, keeping it concise and tidy.

**Comment by JasonDsouza212** [comment]
@Vidip-Ghosh, will you be working on this, or should I assign it to someone else?

**Comment by Vidip-Ghosh** [comment]
> @Vidip-Ghosh, will you be working on this, or should I assign it to someone else?

Yes I will be working

**Comment by VenomFate-619** [comment]
@Vidip-Ghosh post the ss of the changes

**Comment by Vidip-Ghosh** [comment]
> @Vidip-Ghosh post the ss of the changes

Screenshot 2023-06-21 at 16 54 11
Screenshot 2023-06-21 at 16 53 36

**Comment by JasonDsouza212** [comment]
Dont add any color that is out of the theme @Vidip-Ghosh

**Comment by JasonDsouza212** [comment]
The contribution page looks the same as before. Let's try something new and attractive that aligns with our theme.

**Comment by Vidip-Ghosh** [comment]
> The contribution page looks the same as before. Let's try something new and attractive that aligns with our theme.

Can you please suggest some ideas which can be done? Like I added name of the contributors. Any other ideas

**Comment by JasonDsouza212** [comment]
You can create a small card where you can add a title. It should also be responsive, and you can include the necessary information. Just make sure to maintain the color theme and avoid using other colors.

File: src/pages/Community.jsx [hunk 1]
```diff
@@ -38,19 +38,23 @@
...
-                    <img
-                      alt={contributor.login}
-                      className="contributor-image"
-                      loading="lazy"
-                      src={contributor.avatar_url}
-                    />
-                  </a>
+                  <div className='contributor-card'>
+                    <a
+                      className="contributor-anchor"
```
[the remark this answers is not in the document]
Comment (VenomFate-619) [reply src/pages/Community.jsx:43]: fix this.
add this line ``` rel="noopener noreferrer"```

[the remark this answers is not in the document]
Comment (Vidip-Ghosh) [reply src/pages/Community.jsx:44]: Yes I have solved

**Comment by JasonDsouza212** [comment]
Screenshot from 2023-06-23 20-59-25
Screenshot from 2023-06-23 20-59-07

The Navbar has style issues, and the cards have a pure white background. Additionally, the title of contributions is thin and looks odd. You can make the contribution page card text font size a little larger. Similarly, the name should be larger so that it matches the size of the commits, and the border should be slightly bigger as well.

**Comment by Vidip-Ghosh** [comment]
Screenshot 2023-06-24 at 18 37 19
Made font size of contribution page card text a little larger

**Comment by Vidip-Ghosh** [comment]
@JasonDsouza212 @VenomFate-619 Please review this PR

**Comment by JasonDsouza212** [comment]
@Vidip-Ghosh yes ,I will

**Comment by JasonDsouza212** [comment]
The changes made to the contributor section design were not planned or necessary. Please revert them back to the original design as it is in the production application.
Screenshot from 2023-06-30 20-00-07

As you can see, the header is not properly aligned. Additionally, the background of the cards is white, which makes it look improper. Therefore, please set the background of the cards to match the overall white background. Apart from these issues, everything looks good.
Screenshot from 2023-06-30 19-59-46

**Comment by Vidip-Ghosh** [comment]
> The changes made to the contributor section design were not planned or necessary. Please revert them back to the original design as it is in the production application. Screenshot from 2023-06-30 20-00-07
> 
> As you can see, the header is not properly aligned. Additionally, the background of the cards is white, which makes it look improper. Therefore, please set the background of the cards to match the overall white background. Apart from these issues, everything looks good. Screenshot from 2023-06-30 19-59-46

Okay alright

**Comment by Vidip-Ghosh** [comment]
Now check @JasonDsouza212 
Screenshot 2023-06-30 at 22 28 26
Screenshot 2023-06-30 at 22 27 39
Screenshot 2023-06-30 at 22 27 21

**Comment by Vidip-Ghosh** [comment]
@JasonDsouza212 @VenomFate-619 Why its showing the label **title-needs-formatting**?

**Comment by JasonDsouza212** [comment]
Hello, contributors! This is Jason speaking. Due to some workflow issues, we will be closing all the PRs today. Therefore, I kindly request all of you to create new PRs tomorrow, as the currently open ones will be closed by our team. I apologize for any inconvenience caused. Also, please make sure to check the titles when creating the PRs. You can refer to the README.md for more details. Thank you.

**Comment by JasonDsouza212** [comment]
@Vidip-Ghosh, please create a new PR. I believe everything looks good.

**Pull request closed without merging**